extensions/amp-base-carousel/1.0/base-carousel.css.js

```diff
@@ -88,6 +88,111 @@ export const horizontalScroll = {
   scrollSnapTypeX: 'mandatory',
   scrollSnapType: 'x mandatory',
   /* Hide scrollbar */
+  marginBottom: '-20px',
   paddingBottom: '20px',
   overflowY: 'hidden',
 };
+
+/** Styles for pagination indicators. */
+export const paginationContainer = {
```

Comment:
IMHO, this should go into a separate `pagination.css.js` file. There's a very clear and separate stuff going on here. And, nicely, we can simply `import` JS files from each other.